cdhweb/blog/models.py

```diff
@@ -92,6 +91,9 @@ class BlogPost(BasePage, ClusterableModel, PagePreviewDescriptionMixin):
     ]
     promote_panels = Page.promote_panels + [FieldPanel("tags")]
 
+    # index description in addition to body content
+    search_fields = BasePage.search_fields + [index.SearchField("description")]
```

Comment:
Any other fields we should include for other models, like project members on projects? Or does that only matter when add page-specific filters?

Reply:
@thatbudakguy I think you could make a best guess now if you want, for the obvious things (like project team members). If you're not comfortable with that you could draft a list and propose the fields & boosting you suggest, which could be reviewed and implemented later.  It occurs to me that generating a list of project members might not be completely trivial — if it isn't then we should probably track it in a separate issue.

Reply:
this turned out to be super easy using wagtail's `index.RelatedFields`. I didn't play with the boosting at all, but did confirm that it was working — for example, the davila.js project page comes up (on second page of results) when searching "koeser" even though your name isn't in the page body. i think we can revisit boosting (at query time) later if we decide we need to.